apple/internal/partials/debug_symbols.bzl

```diff
@@ -188,6 +211,20 @@ def _debug_symbols_partial_impl(ctx, debug_dependencies = [], debug_outputs_prov
                 transitive = [dsym_bundles],
             )
 
+            include_symbols = defines.bool_value(
```

Comment:
should we continue using defines for this purpose? It doesn't feel like they're really the right way to be doing configuration, vs explicit config settings or target attrs

Reply:
I agree we should look into what we should be doing going forward, but I wouldn't want to block this PR on something that might require a lot of design and thought, not tied to this change. For example, I wouldn't want to introduce configuration hash based output directories (which could happen depending on how we do this).

Reply:
It might not be the right way but I think "define" is how rules_apple is handling its "features" at the moment; for example:  use `--define=apple.package_swift_support` to disable packaging `SwiftSupport` in ipas. You can see more at https://github.com/bazelbuild/rules_apple/blob/master/doc/common_info.md.